src/main/resources/io/jenkins/plugins/coverage/CoveragePullRequestMonitoringPortlet/monitor.jelly

```diff
@@ -0,0 +1,48 @@
+<?jelly escape-by-default='true'?>
+
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+
+    <st:adjunct includes="io.jenkins.plugins.echarts"/>
+    <st:adjunct includes="io.jenkins.plugins.bootstrap5"/>
+
+    <j:set var="targetBranchBuild" value="${it.getComparedBuildLink()}" />
+    <j:if test="${targetBranchBuild.isPresent()}">
+        <j:set var="coverageDiff" value="${it.getDiff()}" />
+        <j:choose>
+            <j:when test="${coverageDiff gt 0}">
```

Comment:
I wonder if we can use the code in `Ratio` to render the percentage.

(In the screenshots it looks like that sometimes for a value of 0.000001 this condition is true, maybe we should compare after rounding)